pkg/network/prometheus.go

```diff
@@ -0,0 +1,66 @@
+package network
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Metric used in monitoring service.
+var (
+	peersConnected = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Help:      "Number of connected peers",
+			Name:      "peers_connected",
+			Namespace: "neogo",
+		},
+	)
+
+	servAndNodeVersion = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Help:      "Server and Node versions",
+			Name:      "serv_node_version",
+			Namespace: "neogo",
+		},
+		[]string{"description", "value"},
+	)
+
+	poolCount = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Help:      "Discovery death pool",
```

Comment:
It's a pool of addresses to connect to, I don't think it has anything to do with death.

Reply:
fixed